src/AvtaleSide/MaalSteg/RedigerMaal/RedigerMaal.tsx

```diff
@@ -32,6 +32,12 @@ class RedigerMaal extends React.Component<Props, State> {
         valgtKategoriFeil: undefined,
     };
 
+    componentDidMount(): void {
+        this.setState({
```

Comment:
Hvorfor sette valgtKategori i componentDidMount? Ser ut til å kun være avhengig av this.props.ledigeMaalkategorier.

Reply:
det er riktig. React castet warning på komponent. at den ikke kunne sette state før komponent før den var mountet, tror jeg. Muligens noe som ikke er helt i sync. kan se mer på det i morgen

Reply:
Ok, får nok den feilen om man kaller `setState` før rendring. Men kan løses ved å sette `this.state = { ... }` lenger opp i komponenten.